### ISSUE TITLE
Allow to create directory from projects tab

### ISSUE THREAD

**Issue by tdudgeon** [InformaticsMatters/squonk2-data-manager-ui#1216]
This was requested by Sygnature and is definitely needed.
The API will need to be updated to allow this.

We should also consider other obvious operations such as:
* rename file/directory
* copy file

**Comment by OliverDudgeon** [comment]
I worry we are trying to fully implement a file manager here. Maybe there is an off the shelf UI component that we just need to attach API end points to?

**Comment by tdudgeon** [comment]
Yes, this could be a slippery slope, but unless there is a general solution then I think we must add the most essential actions such as create dir.
